honeybee_vtk/data.py

```diff
@@ -0,0 +1,161 @@
+"""Data json schema and validation."""
+
+import json
+from json.decoder import JSONDecodeError
+from pathlib import Path
+from typing import Dict, List
+from pydantic import BaseModel, validator, Field
+from pydantic.errors import JsonError
+from .model import Model
+from .types import AcceptedValues
+
+
+class Data(BaseModel):
+    """COnfig for data."""
+
+    name: str = Field(
+        description='Name to be give to data. Example, "Daylight-Factor".'
+    )
+
+    object_name: str = Field(
```

Comment:
```suggestion
    object_name: AcceptedValues = Field(
```

This will take care of validation for you. See the first example on manual page: https://pydantic-docs.helpmanual.io/usage/schema/

and object name is very generic. I wouldn't know what are you referring to here without checking the validator. Is it object type in the model?

Reply:
I am no longer using Enum for these values, so I will keep the validator for this one. I changed the name to "object_type" though.